kustomize/internal/commands/edit/listbuiltin/listbuiltin.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package listbuiltin
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/konfig"
+)
+
+// NewCmdListBuiltinPlugin return an instance of list-builtin-plugin
+// subcommand
+func NewCmdListBuiltinPlugin() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-builtin-plugin",
```

Comment:
Please add `alpha-` here.

We don't want to commit to this UX for all time - there are other kustomize build query methods we could add and need more time to consider how to arrange the commands.

Putting alpha here is a signal that the UX will change.